### PR TITLE
core(main-thread-tasks): use toplevel tasks in main thread audit

### PR DESCRIPTION
**Pull request by patrickhulce**
**Summary**
Typo in the `main-thread-tasks` audit, I didn't get around to surfacing this yet, so it shouldn't have hurt anything. I think I was originally going for first inner child of toplevel task so we'd actually have a type, but then again I wasn't using the results yet, so 🤷‍♂️ 

**Related Issues/PRs**
#7700

### CODE REVIEW

**Comment by patrickhulce** [comment]
cc @alekseykulikov

File: lighthouse-core/audits/main-thread-tasks.js [hunk 1]
```diff
@@ -32,18 +32,16 @@ class MainThreadTasks extends Audit {
     const tasks = await MainThreadTasksComputed.request(trace, context);
 
     const results = tasks
-      .filter(task => task.duration > 5 && task.parent)
+      .filter(task => task.duration > 5 && !task.parent)
```
Comment (brendankenny) [inline lighthouse-core/audits/main-thread-tasks.js]: maybe add a comment that top level === no parent here? Easy to miss the point of the filter though it's obvious once you realize :)

Comment (patrickhulce) [reply lighthouse-core/audits/main-thread-tasks.js]: yeah good call, I mean I missed it the first time around 😆

**Comment by alekseykulikov** [comment]
LGTM!

**Pull request merged**